src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -156,7 +166,6 @@ protected Task Stop()
             RequestContext<object> requestContext)
         {
             // Allow the implementor to shut down gracefully
-            await this.Stop();
```

Comment:
This looks like we are simply ignoring the ShutdownRequest message?  Does this mean we are really only processing the ExitNotification message in order to stop the lang server?

Reply:
That's correct. The `shutdown` message is not supposed to actually do things that would stop  the process.

It's the message that gets sent to inform the language server that a shutdown is occurring and that you should expect an `exit` message soon.